homeassistant/components/coinbase/sensor.py

```diff
@@ -105,6 +105,7 @@ def __init__(self, coinbase_data, currency):
                     API_ACCOUNT_CURRENCY
                 ]
                 break
+        self._attr_state_class = STATE_CLASS_MEASUREMENT
```

Comment:
This is an account balance, this is not a measurement, but a total? 

IMHO the state class is wrong on this one.

Reply:
@frenck done in #59404. I do think measurement is correct for exchange rate as it isn't anything the user has/had and is a dimensionless ratio anyway

Reply:
Yep, an exchange rate is a measurement 👍